docs-v2/integrations/all/acuity-scheduling.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Acuity Scheduling
+sidebarTitle: Acuity Scheduling
+---
+
+API configuration: [`acuity-scheduling`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Auth (OAuth)](/integrate/guides/authorize-an-api) | ✅ |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | ✅ |
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+- [How to register an Application](https://acuityscheduling.com/oauth2/register)
+- [OAuth-related docs](https://developers.acuityscheduling.com/docs/oauth2)
+- [API](https://developers.acuityscheduling.com/reference/quick-start)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+## API gotchas
+
+- Acuity uses OAuth for authentication. Ensure you have the correct client ID and secret.
+- Be aware of rate limits and plan your API calls accordingly.
```

Comment:
- Please add the link to the resource; `<Note>Add Getting Started links and Gotchas by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/acuity-scheduling.mdx)</Note>`

- Would be nice to mention;  `Acuity Scheduling enforces a general rate limit of 10 requests a second and 20 concurrent connections from an IP.`

Reply:
added!